charts/gsp-cluster/templates/05-namespaces/namespace.yaml

```diff
@@ -0,0 +1,43 @@
+{{- range .Values.namespaces }}
+{{- range .resources }}
+---
+{{ tpl (toYaml .) $ }}
+{{- end }}
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
Maybe this should be moved out of here and we define it in the config namespace directory (ie gsp-teams) ... this would mean a bit of duplication, but would give us the flexibility to mess with annotations correctly

Reply:
I agree. This belongs in gsp-teams.